ext/standard/tests/math/clamp_basic2.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+Test clamp() function : basic functionality
+--FILE--
+<?php
+echo "*** Testing clamp() : basic functionality - using floats ***\n";
+
+var_dump(clamp(0.0, 1.0, 2.0));
+var_dump(clamp(1.5, 1.0, 2.0));
+var_dump(clamp(2.0, 1.0, 2.5));
+var_dump(clamp(3.0, 1.0, 2.5));
+var_dump(clamp(0, 3.0, 3.5));
```

Comment:
need more testing, integer, mixed types (integer/float), INF, NAN, ...

Reply:
Wasn't really sure what to test so copied some values from the `abs` test suite, that should be OK to do right? Will add more tomorrow with max 64/32bit values.